File: Printing-Pattern-Programs-main/PythonPatternPrograms/SymbolPatterns/Pattern 43.py
n = 9  # size odd
c1 = (n - 1) // 2  # cond
c2 = 3 * n // 2 - 1  # cond

for x in range(0, n):
    for y in range(0, n):
        if (x + y == c1 or x - y == c1 or y - x == c1 or x + y == c2 or x == c1 or y == c1):
            print("* ", end="")  # *space
        else:
            print("  ", end="")  # two spaces
    print()for x in range(1, 6):
    for y in range(65, 70):
        print(chr(y), end="")
    print()

File: Printing-Pattern-Programs-main/JavaPatternPrograms/SymbolPatterns/Pattern 43.java
 class Pattern_129
 {
     public static void main(String args[])
     {
      int i;
      int j;
 
      int n = 9; //size (odd)
 
      int c1 = (n - 1) / 2; //cond
 
      int c2 = 3 * n / 2 - 1; //cond
 
      for (i = 0;i < n;i++)
      {
       for (j = 0;j < n;j++)
       {
         if (i + j == c1 || i - j == c1 || j - i == c1 || i + j == 
       c2 || i == c1 || j == c1)
         {
          System.out.print("* ");
         }
         else
         {
          System.out.print("  ");
         }
       }
       System.out.println("");
      }
     }
 }